src/lib/server.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const http = require('http');
+const cowsay = require('cowsay');
+const logger = require('./logger');
+const requestParser = require('./request-parser');
+
+const app = http.createServer((request, response) => {
+  logger.log(logger.INFO, 'New request');
+  logger.log(logger.INFO, `METHOD: ${request.method}`);
+  logger.log(logger.INFO, `ROUTE: ${request.url}`);
+
+  return requestParser.parseASYNC(request)
+    .then((parsedRequest) => {
+      if (parsedRequest.method === 'GET' && parsedRequest.url === '/') {
+        response.writeHead(200, { 'Content-Type': 'text/html' });
+
+        response.write(`<!DOCTYPE html> 
+                        <head>Here's the head</head>
+                        <body>Dogs are better than cats!!</body>
+                        </html>`);
+        logger.log(logger.INFO, 'Responding back with 200 status code and html document');
+        response.end();
+        return undefined;
+      } if (parsedRequest.method === 'POST' && parsedRequest.url === '/api/cowsay') {
+        response.writeHead(200, { 'Content-Type': 'application/json' });
+        response.write(cowsay.say({
+          text: "I'm a moooodule",
```

Comment:
your cow's word aren't being updated when a user types in their own text.  How could we access the text information someone enters into the command line?